reactioncommerce/reaction
# Allow users to disable plugins via config

Resolves #NA
Impact: **minor**  
Type: **feature**

## Issue
There is no good and easy way to disable a plugin through configuration

## Solution
Keep a list of disabled plugins and don't load/import any of those

## Breaking changes
None

## Testing
1. Start Reaction and observe no problems
1. Edit `.reaction/pluginConfigs` and add `search-mongo` to the array for `disabledPlugins`.
1. Restart Reaction
1. Observe that the mongo search is disabled
